support/ref-docs/src/arrays.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Joins (or concatenates) all arrays passed to this function into one
+ * continuous array.
+ */
+export function joinArrs<T>(...arrs: readonly (readonly T[])[]): T[] {
```

Comment:
the [`Array.prototype.flat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) method might be a good substitute for this helper. for example, `joinArrs([a, b], [c, d, e])` is the same as `[[a, b], [c, d, e]].flat()`.

it's only available in Node 11.0 and above, but then i think including a polyfill with the same API is better than using this helper. see: https://www.npmjs.com/package/array.prototype.flat